src/controllers/starting.new.controller.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from "express";
+
+import * as config from "../config";
+import { logger } from "../utils/logger";
+
+export const get = (req: Request, res: Response, _: NextFunction) => {
+  logger.debugRequest(req, `GET ${config.STARTING_NEW_PAGE}`);
+
+  return res.render(config.STARTING_NEW_PAGE, {
+    backLinkUrl: config.LANDING_PAGE_URL,
+    templateName: config.STARTING_NEW_PAGE
+  });
+};
+
+export const post = (req: Request, res: Response, _: NextFunction) => {
+  logger.debugRequest(req, `POST ${config.STARTING_NEW_PAGE}`);
+
+  if (req.body["starting_new"] === 'yes') {
```

Comment:
i think we should rename the html field on the body to 'continue_saved_application'
as if "starting_new" = yes then we are continuing a saved application
if "starting_new"= no then we are actually starting a new application